study.py

```diff
@@ -27,14 +27,16 @@ def study():
     simCnt = 0
 
     now = datetime.now()
+    if config.deterministic:
+        random.seed("DASsimulator")
```

Comment:
For the deterministic branch, I think it would be better to be able to configure the seed, and then maybe combine that with the shape to get a deterministic seed per run.

Reply:
Do you mean to initialize the seed for each shape?

Reply:
Yes, the idea would be that the configured seed and the shape (together with the run number) determines the RNG used, making individual runs reproducible. This is useful for debugging or extending results without re-running the whole study.